mods/common/chrome/lobby-players.yaml

```diff
@@ -252,7 +252,7 @@ Container@LOBBY_PLAYER_BIN:
 									Height: 25
 									Text: label-lobby-players-faction
 						Label@TEAM:
-							X: 420
+							X: 410
 							Width: 42
```

Comment:
Confirmed that changing this to `Width: 43` fixes the alignment for both cases.
<img width="176" src="https://github.com/user-attachments/assets/98d245f8-acd5-4edc-9a13-fa42a1f44335" /> <img width="256" src="https://github.com/user-attachments/assets/1f0cfbe0-7c0b-4aab-b609-52f2c8fa7801" />

Reply:
adding +1 width seems to have fixed it in all mods